iOS-Play/iOS-Play/Controller/ViewModel/AppCollection/CollectionViewCell.swift

```diff
@@ -0,0 +1,83 @@
+//
+//  CollectionViewCell.swift
+//  iOS-Play
+//
+//  Created by Quang Nguyen on 4/3/21.
+//
+
+import Foundation
+import UIKit
+
+final class CollectionViewCell: UICollectionViewCell {
+    
+    private let appImage = UIImageView().then {
+        $0.layer.cornerRadius = 20
+        $0.clipsToBounds = true
+        $0.contentMode = .scaleAspectFill
+    }
+    
+    private let nameAppLable = UILabel().then {
+        $0.numberOfLines = 0
+        $0.contentMode = .topLeft
+        $0.font = FontFamily.Arvo.bold.font(size: 14)
+        $0.textColor = .darkGray
+    }
+    
+    private let nameAuthorLable = UILabel().then {
+        $0.numberOfLines = 0
+        $0.contentMode = .topLeft
+        $0.font = FontFamily.Arvo.regular.font(size: 14)
+        $0.textColor = .darkGray
+    }
+    
+    private let releaseLable = UILabel().then {
+        $0.numberOfLines = 0
+        $0.contentMode = .topLeft
+        $0.font = FontFamily.Arvo.regular.font(size: 14)
+        $0.textColor = .darkGray
+    }
```

Comment:
view em dùng Storyboard và xib đi em, làm trong swift thì file nở ra to

Reply:
Em muốn code chay hết ạ, vì e thấy dễ quản lí và nhanh hơn dùng xib ạ, nên e được giữ lại không ạ